fabric/bin/lib/common_utils.sh

```diff
@@ -59,3 +59,20 @@ try_r() {
     echo $RESPONSE
 }
 
+# Check the Response returned to validate expected result
+check_result() {
```

Comment:
This function is in `common_ledger.sh`.
Check out https://github.com/hyperledger-labs/fabric-private-chaincode/blob/06fc10f237669a9d63bf0714b314b4e3c1edf8da/fabric/bin/lib/common_ledger.sh#L169